src/AutoRest.CSharp.V3/Input/SchemaUsageProvider.cs

```diff
@@ -35,24 +35,34 @@ public SchemaUsageProvider(CodeModel codeModel)
                         Apply(operationResponse.ResponseSchema, SchemaTypeUsage.Error | SchemaTypeUsage.Output);
                     }
 
+                    foreach (var parameter in operation.Parameters)
```

Comment:
Do we use these?

Reply:
Discussed offline: we also use it [here](https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp.V3/Output/Models/RestClient.cs#L40).